src/Command/TwoStepVerificationCommand.php

```diff
@@ -16,34 +16,23 @@
 use FOS\UserBundle\Model\UserManagerInterface;
 use Sonata\UserBundle\GoogleAuthenticator\Helper;
 use Sonata\UserBundle\Model\UserInterface;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
-/**
- * NEXT_MAJOR: stop extending ContainerAwareCommand.
- */
-class TwoStepVerificationCommand extends ContainerAwareCommand
+class TwoStepVerificationCommand extends Command
 {
-    /**
-     * @var ?Helper
-     */
-    private $helper;
+    protected static $defaultName = 'sonata:user:two-step-verification';
 
-    /**
-     * @var ?UserManagerInterface
-     */
+    private $helper;
     private $userManager;
 
-    /**
-     * NEXT_MAJOR: make $helper and $userManager mandatory (but still nullable).
-     */
     public function __construct(
-        ?string $name,
-        ?Helper $helper = null,
-        ?UserManagerInterface $userManager = null
+        ?Helper $helper,
+        ?UserManagerInterface $userManager,
+        ?string $name = null
```

Comment:
I don't think the comment asked to change the order of the param.

And IMHO, if we're throwing an exception:
```
if (null === $this->helper || null === $this->userManager) {
    throw new \RuntimeException('You need to inject Helper and UserManager.');
}
```
I don't understand why we should allow `null` here.

Reply:
@greg0ire You made this commit: https://github.com/sonata-project/SonataUserBundle/commit/27de0075caa18dd40987733258b94cf7acb122cb

Maybe you can explain ?

Reply:
- if we're accepting `?` then it can be `null`, But I need those classes to this code work, But if it's required for this code work then it shouldn't be null! :confused: 

- I changed the orders because optional parameters should be in the end of parameters list :confused: (But I guess if put `?$name`, Symfony will pass a null value to there ... then I don't need to put a default value ...., I'll test it)

Reply:
> * if we're accepting `?` then it can be `null`, But I need those classes to this code work, But if it's required for this code work then it shouldn't be null! 😕

I agree.
Unless someone has an argument against I would use `Helper $helper` and `UserManagerInterface $userManager`.

Reply:
> Maybe you can explain ?

@VincentLanglet  the end goal of 27de007 is to push people to use DI. `null` should not be allowed.